src/marshmallow/fields.py

```diff
@@ -416,7 +417,13 @@ class Nested(Field):
     default_error_messages = {"type": "Invalid type."}
 
     def __init__(
-        self, nested, *, default=missing_, exclude=tuple(), only=None, **kwargs
+        self,
+        nested: SchemaABC,
```

Comment:
Can probably just be `Schema`. Users shouldn't really deal with `SchemaABC` in practice.